README.md

```diff
@@ -93,8 +93,6 @@ On the modern web, sites span multiple domains and many sites are owned & operat
     -   uber.com, ubereats.com
 -   Country-specific domains to enable localization 
     -   google.co.in, google.co.uk
```

Comment:
You took ccTLD out below. What basis does this have?  "Associated"?

Reply:
The ccTLD subset wasn't completely removed. It wasn't quite accurate to call it a subset, so now it's an orthogonal mechanism that is described separately; see lines 213-232 in the new version.